code/admin/language/en-GB/en-GB.com_api.ini

```diff
@@ -80,8 +80,10 @@ COM_API_KEYS_PER_HOUR="Per hour"
 
 ; Config
 COM_API_CONFIG_LOG_LBL="Log Requests"
+COM_API_CONFIG_ALLOW_CORS_LBL="Allow Cross Origin Requests"
+COM_API_CONFIG_ALLOW_CORS_DESC="Choose which method is allowed for Cross Origin Requests."

```

Comment:
Change this to 

Comma separated list of HTTP methods to allow CORS support for. Put an asterisk (*) to allow CORS requests for all methods.